bin/update-releases.sh

```diff
@@ -56,25 +73,42 @@ do
 done
 
 update_submodule diego-release "${DIEGO_RELEASE}" src
-update_submodule cflinuxfs2-rootfs-release "${CFLINUXFS2_ROOTFS_RELEASE}" src
+update_submodule cflinuxfs2-rootfs-release "${CFLINUXFS2_OLD_ROOTFS_RELEASE}" src
+update_submodule cflinuxfs2-release "${CFLINUXFS2_ROOTFS_RELEASE}" src
 update_submodule garden-runc-release "${GARDEN_RUNC_RELEASE}" src
 
 CF_RELEASE_VERSION_INFO=$(curl --silent "https://api.github.com/repos/cloudfoundry/cf-release/contents/src?ref=${CF_RELEASE}")
 
+echo > ${GIT_ROOT}/_work/CF_RELEASE_VERSION_INFO "${CF_RELEASE_VERSION_INFO}"
+echo > ${GIT_ROOT}/_work/SUBMODULE_REFERENCES    ""
+
 get_submodule_ref () {
 	version_info=${1}
 	release=${2}
+
+	case $release in
+	    *-buildpack-*)
+		where=$(cd ${GIT_ROOT}/src/buildpacks/$release ; git remote -v | grep fetch | awk '{ print $2 }' | uniq)
+		;;
+	    *)
+		where=$(cd ${GIT_ROOT}/src/$release ; git remote -v | grep fetch| awk '{ print $2 }' | uniq)
+		;;
+	esac
+
+    echo "${release}" = $(echo "${version_info}" | jq -r ".[] | select(.name == \"${release}\") | .sha") '@' $where \
+        >> ${GIT_ROOT}/_work/SUBMODULE_REFERENCES
+
 	echo "${version_info}" | jq -r ".[] | select(.name == \"${release}\") | .sha"
 }
 
-GO_BUILDPACK_RELEASE=$(get_submodule_ref "${CF_RELEASE_VERSION_INFO}" go-buildpack-release)
-BINARY_BUILDPACK_RELEASE=$(get_submodule_ref "${CF_RELEASE_VERSION_INFO}" binary-buildpack-release)
-NODEJS_BUILDPACK_RELEASE=$(get_submodule_ref "${CF_RELEASE_VERSION_INFO}" nodejs-buildpack-release)
-RUBY_BUILDPACK_RELEASE=$(get_submodule_ref "${CF_RELEASE_VERSION_INFO}" ruby-buildpack-release)
-PHP_BUILDPACK_RELEASE=$(get_submodule_ref "${CF_RELEASE_VERSION_INFO}" php-buildpack-release)
-PYTHON_BUILDPACK_RELEASE=$(get_submodule_ref "${CF_RELEASE_VERSION_INFO}" python-buildpack-release)
-STATICFILE_BUILDPACK_RELEASE=$(get_submodule_ref "${CF_RELEASE_VERSION_INFO}" staticfile-buildpack-release)
-JAVA_BUILDPACK_RELEASE=$(get_submodule_ref "${CF_RELEASE_VERSION_INFO}" java-buildpack-release)
+# GO_BUILDPACK_RELEASE=$(get_submodule_ref "${CF_RELEASE_VERSION_INFO}" go-buildpack-release)
```

Comment:
The buildpacks are our own forks, they don't get bumped anymore, or rather, bumped independently of the scf core here.